py/examples/dropdown.py

```diff
@@ -34,7 +34,7 @@ async def serve(q: Q):
             ui.dropdown(name='dropdown_disabled', label='Pick one (Disabled)', value='B', choices=choices,
                         disabled=True),
             ui.dropdown(name='dropdown_dialog', label='Pick multiple in dialog (>100 choices)', values=['1'],
-                        required=True, choices=choices_dialog),
+                        required=True, choices=choices_dialog, placeholder='Select values...'),
```

Comment:
Placeholder should not replace a label. See https://www.nngroup.com/articles/form-design-placeholders/ for more UX insights.

Reply:
Sounds like it can cause confusion indeed. 

From your link:
> [placeholder] is reserved for supplementary information.

In this case, should we provide a placeholder for these examples?

Reply:
I cannot think of a valid scenario (but maybe there is) for a dropdown with a placeholder. Placeholder serves to show example values - primary case or some extra info needed, but with dropdown example / extra info are not needed since options are constrained. It's a whole different story for textboxes/comboboxes where a user is allowed to specify arbitrary strings.

Let's not use placeholder in this dropdown example, but make sure it is displayed correctly if specified.

Reply:
Agreed. 
The latest changes don't specify `placeholder` for examples and display it correctly if specified.
Besides that, I removed a hardcoded placeholder "Select ...", it was being displayed in the textbox after the user deselects all options.